src/utils/entity.utils.ts

```diff
@@ -2,7 +2,7 @@
  * A key-value model that keeps an order of the elements by
  * the input sort function.
  *
- * Insipration by NgRx/EntityState: https://github.com/ngrx/platform
+ * Inspiration by NgRx/EntityState: https://github.com/ngrx/platform
```

Comment:
Actually this should be `Inspired by` or `Inspiration from`...

Reply:
True, or even `Inspired with`. Feel free to fix it! :)